PACKAGING.md

```diff
@@ -662,3 +664,32 @@ You can even specify the path to another folder, if for example you've got anoth
 fpm package in the same repository. Like this: `helloff = { path = "helloff" }`.
 Note that you should *not* specify paths outside of your repository, or things
 won't work for your users.
+
+### Custom Build Scripts
+
+If there is something special about your library that makes fpm unable to build
+it, you can provide your own build script. fpm will then simply call your
+build script to build the library.
+
+To specify a build script to be used, put it in the library section of your
+`fpm.toml` file, like:
+
+```toml
+[library]
+source-dir="src"
+build-script="my_build_script"
+```
+
+fpm will set the following environment variables to specify some parameters to
+the build script.
+
+* `FC` - The Fortran compiler to be used
+* `FFLAGS` - The flags that should be passed to the Fortran compiler
+* `BUILD_DIR` - Where the compiled files should be placed
+* `INCLUDE_DIRS` - The folders where any dependencies can be found
```

Comment:
The trailing 'S' implies multiple folders can be specified - is this done in a system-dependent manner using colon/semicolon separators? If so, it's worth mentioning that here. (I had thought multiple include paths had to be specified with multiple `-I` flags.)

Reply:
They are space separated. So yes, that should be mentioned here, and it is then necessary for your build script to turn this list into the appropriate flags for the compiler(s). I'll add this info.